frontend/src/features/admin-form/settings/SettingsWorkflowPage.tsx

```diff
@@ -0,0 +1,51 @@
+import { Link, Text, Wrap } from '@chakra-ui/react'
+
+import { FormResponseMode } from '~shared/types'
+
+import { GUIDE_TWILIO } from '~constants/links'
+
+import { CategoryHeader } from './components/CategoryHeader'
+import { WorkflowDetailsInput } from './components/WorkflowSettingsSection/WorkflowDetailsInput'
+import { WorkflowUnsupportedMsg } from './components/WorkflowSettingsSection/WorkflowUnsupportedMsg'
+import { useAdminFormSettings } from './queries'
+
+export const SettingsWorkflowPage = (): JSX.Element => {
+  const { data: settings, isLoading } = useAdminFormSettings()
+
+  // Workflow is only supported in multirespondent mode; show message if form response mode is not multirespondent
+  if (
+    !isLoading &&
+    settings?.responseMode !== FormResponseMode.Multirespondent
+  ) {
+    return <WorkflowUnsupportedMsg />
+  }
+  return (
+    <>
+      <Wrap
+        shouldWrapChildren
+        justify="space-between"
+        align="center"
+        mb="2.5rem"
+      >
+        <CategoryHeader mb={0} mr="2rem">
+          Workflow
+        </CategoryHeader>
+      </Wrap>
+      {settings?.responseMode === FormResponseMode.Multirespondent ? (
```

Comment:
this check can be extracted out above to make the render function simpler to read

Reply:
[23a4e77](https://github.com/opengovsg/FormSG/pull/6968/commits/23a4e77e7575af3cb31457a256354a2a5ad16cb3)